pennylane/devices/qubit/simulate.py

```diff
@@ -35,6 +58,7 @@ def simulate(circuit: qml.tape.QuantumScript, rng=None, debugger=None) -> Result
             seed-like parameter matching that of ``seed`` for ``numpy.random.default_rng``.
             If no value is provided, a default RNG will be used.
         debugger (._Debugger): The debugger to use
+        return_final_state (bool): Whether to return the final state in addition to the results
 
     Returns:
         tuple(TensorLike): The results of the simulation
```

Comment:
Might be worth adding a bit more information about what the return looks like if `return_final_state` is true.  We should be able to determine what is returned purely by looking at the documentation.

Reply:
Same as above, but noted 👍